samples/core/output_a_directory/output_a_directory.py

```diff
@@ -0,0 +1,29 @@
+import kfp
+from kfp.components import create_component_from_func, InputPath, OutputPath
+
+@create_component_from_func
```

Comment:
It would probably be useful to add a few lines of comment (or a mini REAMDE) e.g. 'this example shows how to..."  as I suspect if someone has not tried to do this yet they might not understand the gist of the example.

Reply:
Added the explanation.